gdpr/anonymizers/model_anonymizers.py

```diff
@@ -292,36 +299,80 @@ def anonymize_qs(self, qs: QuerySet) -> None:
     def deanonymize_qs(self, qs: QuerySet) -> None:
         raise NotImplementedError()
 
+    def get_anonymizer_or_none(self, name):
+        return self.anonymizers.get(name, None)
```

Comment:
The second argument "None" is not needed -  this is default. In that case, this method is just useless wrapper.